### ISSUE TITLE
Support for factory-boy 3+

### ISSUE THREAD

**Issue by michael0liver** [mimesis-lab/mimesis-factory#183]
Hey factory-boy have released version 3 but it looks like mimesis-factory depends on factory-boy (>=2.11,<3.0). Could we get this supported?

Much thanks for a great library!

**Comment by sobolevn** [comment]
PRs are welcome!

**Comment by michael0liver** [comment]
> PRs are welcome!

I will try :)

**Comment by michael0liver** [comment]
Strange issue, this project clearly defines `factory-boy = ">=2.11,<4.0"` inside the pyproject.toml directory. But when I try to install with in my own project poetry I get:
```bash
[SolverProblemError]
Because mimesis-factory (1.1.0) depends on factory-boy (>=2.11,<3.0)
 and no versions of mimesis-factory match >1.1.0,<2.0.0, mimesis-factory (>=1.1.0,<2.0.0) requires factory-boy (>=2.11,<3.0).
So, because dacp depends on both mimesis-factory (^1.1.0) and factory-boy (^3), version solving failed.
```

Very strange because when I clone this repo and run `poetry install` it actually installs factory-boy v3.0.1

**Comment by sobolevn** [comment]
Checkout this line https://github.com/mimesis-lab/mimesis-factory/blob/1.1.0/pyproject.toml#L40

**Comment by michael0liver** [comment]
> Checkout this line https://github.com/mimesis-lab/mimesis-factory/blob/1.1.0/pyproject.toml#L40

I was looking at the master branch (where it has been bumped). Looks like dependabot bumped the version in mimesis-lab/mimesis-factory/pull/178. How do we tag/initiate a release?

**Comment by sobolevn** [comment]
Manually only. I guess we can start with adding different `factoryboy` version to the CI, like here: https://github.com/sobolevn/django-split-settings/blob/master/.travis.yml#L9

**Comment by michael0liver** [comment]
Maybe I don't understand but I don't think it should effect anything since the current CI is still passing with the newest `factory-boy` version: https://travis-ci.com/github/mimesis-lab/mimesis-factory/jobs/376062661#L302

**Comment by michael0liver** [comment]
Can we trigger a release?

**Comment by sobolevn** [comment]
Yes, will do today!

**Comment by sobolevn** [comment]
Released `1.2.0`
